manim/scene/graph_scene.py

```diff
@@ -260,25 +258,26 @@ def setup_axes(self, animate=False):
         self.default_graph_colors = it.cycle(self.default_graph_colors)
 
     def coords_to_point(self, x, y):
-        """
-        The graph is smaller than the scene.
-        Because of this, coordinates in the scene don't map
-        to coordinates on the graph.
-        This method returns a scaled coordinate for the graph,
-        given cartesian coordinates that correspond to the scene..
+        """The graph is smaller than the scene.
+
+        Because of this, coordinates in the graph don't map
+        to coordinates on the scene.
+
+        This method returns a scaled coordinate for the scene,
+        given coordinates that correspond to the graph.
 
         Parameters
         ----------
         x : int, float
```

Comment:
I know this isn't something you have written, but while this PR touches this method, we may as well improve the rest of the class/method. Please read our [documentation guidelines](https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP)#types).
```suggestion
         x : :class:`float`
```

Reply:
Something about my system configuration (working on Windows10) is preventing me from running make in manim/docs, so I've done the best I can without being able to see the rendered documentation.

Reply:
For the docs, you should have a look at https://github.com/ManimCommunity/manim/wiki/Documentation-guidelines-(WIP)#types, also you can get a preview using RTD. https://manimce--1076.org.readthedocs.build/en/1076/